### PR TITLE
make spack ci run use arbor version information

### PR DESCRIPTION
**Pull request by bcumming**

### CODE REVIEW

**Review by schmitts — COMMENTED** [review]
OK as soon as https://github.com/schmitts/arbor/actions/runs/1082718859 is green.

File: scripts/build_spack_package.sh [hunk 1]
```diff
@@ -54,4 +54,7 @@ spack repo add $SPACK_CUSTOM_REPO
 
 cp $ARBOR_DIR/spack/package.py $SPACK_CUSTOM_REPO/packages/arbor
 cd $ARBOR_DIR
-spack dev-build arbor@with-package-from-repo
+
+ARBOR_VERSION=$(cat "$ARBOR_DIR/VERSION")
+GIT_HASH=$(git log -1 --pretty=format:'%H')
```
Comment (schmitts) [inline scripts/build_spack_package.sh:59]: we are in ARBOR_DIR which is not a git repo

**Comment by schmitts** [comment]
@bcumming Drop this PR as it is included in #1619?

**Comment by bcumming** [comment]
Yes, indeed, thanks!

**Pull request closed without merging**